calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,147 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI. Because the scanner is not running continuously, it may take some time for results to display in the dashboard, especially if you have many images or large images.
+
+## Before you begin
+
+**Required**
+
+- You must have valid registry credentials
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Google GCR 
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Setting up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a docker image. Simply run `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4` in order to retrieve it.
+
+### Setting up registry scanner
+
+#### Credentials
+
+The registry scanner supports scanning all registries as long as the docker credentials are provided.
```

Comment:
```suggestion
The registry scanner supports scanning all registries as long as the Docker credentials are provided.
```

Reply:
Could we change the wording from "... scanning all registries ..." to "... scanning a registry ..."? 

I just want to avoid implying that a single running instance of the registry scanner can scan 2+ registries in one go. It cannot at this point in time.

Reply:
Agree. Fixed.